### PR TITLE
Added in browserify

### PR DESCRIPTION
**Pull request by mrbusysky**
Also made it so the file minifies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +23,9 @@
   ],
   "dependencies": {
     "axios": "^0.21.0",
+    "browserify": "^17.0.0",
```
Comment (sno2) [inline package.json:26]: This should only a dev dependency, right?

File: package.json [hunk 2]
```diff
@@ -22,7 +23,9 @@
...
     "core-js": "^3.7.0",
+    "tsify": "^5.0.2",
```
Comment (sno2) [inline package.json:28]: This should only a dev dependency, right?

**Review by sno2 — CHANGES_REQUESTED** [review]
Looks good, please read my comments on tsify and browserify being regular dependencies.

**Review by mrbusysky — COMMENTED** [review]
If your building an npm you are correct.

File: package.json (hunk 2 above)
Comment (mrbusysky) [reply package.json:28]: if you are making an npm then yes correct.

File: package.json (hunk 1 above)
Comment (mrbusysky) [reply package.json:26]: If your building an npm you are correct.

**Review by sno2 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**